modules/ROOT/pages/7.5-release-notes.adoc

```diff
@@ -69,6 +69,20 @@ The {productname} {release-version} release includes an accompanying release of
 
 For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].
 
+=== Comments
+
+The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** Premium plugin includes the following fix.
```

Comment:
I feel like there might be more comments fixes than just this, but I'm not sure where they're at yet.

Reply:
Yes, most are in review and qa still.